src/cljdoc/server/pedestal.clj

```diff
@@ -299,6 +305,7 @@
    {:keys [route-name] :as route}]
   (->> (case route-name
          :home       [{:name ::home :enter #(ok-html! % (render-home/home))}]
+         :sitemap    [{:name ::sitemap :enter #(ok-xml! % (sitemap/sitemap storage))}]
```

Comment:
This is going to be relatively expensive to compute and render, can we think about some form of caching here? At the least, we should set some caching headers, but I think it could also be good to look at caching in-memory the sitemap for some number of minutes, maybe 60? Another idea would be to save it to disk at some location and serve it from there. You could even have nginx serve it up, and have cljdoc update it in a background process.